src/Compilers/Core/Portable/InternalUtilities/RoslynExperiments.cs

```diff
@@ -12,11 +12,9 @@ internal static class RoslynExperiments
     internal const string NullableDisabledSemanticModel = "RSEXPERIMENTAL001";
     internal const string NullableDisabledSemanticModel_Url = "https://github.com/dotnet/roslyn/issues/70609";
 
-    internal const string SyntaxTokenParser = "RSEXPERIMENTAL003";
-    internal const string SyntaxTokenParser_Url = "https://github.com/dotnet/roslyn/issues/73002";
```

Comment:
Not related to this PR: should we consider removing the experimental flag from remaining APIs too?

Reply:
We certainly could